maps/GWDM/2.0/HDRUK/3.0.0/translation.jsonata

```diff
@@ -1,7 +1,7 @@
 (
 /*function to enforce any arrays to be strings */
 $to_str := function($x){
-  $type($x)="array" ? $join($x,',') : $x
+  $type($x)="array" ? $join($x,';,;') : $x
```

Comment:
`;,;` is the delimiter some previous versions of GWDM has used to concatenate arrays into strings.  Using it here fixes some data losses during translation but I'm not confident that changing this won't break something else.  Reviewers, please advise!

Reply:
yehhh we need a review of this perhaps! you're right, the origin is here from mk1 `datasetv1` to hdruk `2.1.2`:
https://github.com/HDRUK/traser-mapping-files/blob/master/maps/HDRUK/2.1.2/HDRUK/datasetv2/translation.jsonata#L19-L28

I think we should consistently use `;,;` as a delimiter in the GWDM, just may need a ticket to check some of the other mapping files if there is any joins that are not using that...

annoyingly jsonata is not very well setup to share functions between files - i guess we used it way more than we ever thought!

Reply:
I have created a ticket to review the delimiters so it doesn't get forgotten.